### ISSUE TITLE
Should not lint on list passed to function

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#19]
## box.linters version

0.0.0.9006

## Source code to lint

```R
some_function <- function(x) {
  x$element
}

some_function(list(element = \"a\"))
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Expected result

Should not lint on list element reference `x$element`.
